### ISSUE TITLE
Consider not using LBFO. 

### ISSUE THREAD

**Issue by jaromirk** [nmdange2/HyperV_DSC#2]
Just add VMSwitch and configure RDMA vNICs ... No-one cares about LBFO anymore (all focus is now in VMSwitch).
https://github.com/nmdange2/HyperV_DSC/blob/master/DSC/Start-FirstRun.ps1#L144-L161

**Comment by nmdange2** [comment]
I've heard that but I'm not a big fan of the idea of having to install Hyper-V on a Scale-Out File Server cluster that will not be running virtual machines just to get switch-embedded teaming.  I think I can update the configuration to allow either option, though.

**Comment by jaromirk** [comment]
LBFO does not support RDMA, RSS .. So it will be slow as hell. You need it as you need to set VLAN for RoCE RDMA NICs. If using iWARP, you can leave it as is (without SET). But you will have 2 IP addresses. Some agents might not like it.

**Comment by nmdange2** [comment]
Yes that I do know.  Right now my SOFS clusters have two NICs, one non-RDMA NIC that uses an LBFO team for the public/management interface, and then an RDMA NIC with no teaming that is for the SMB traffic.

**Comment by eponerine** [comment]
There is really no risk of installing Hyper-V (unless its a DC or you plan on using another Hypervisor)
